src/simtools/visualization/plot_array_layout.py

```diff
@@ -23,6 +40,10 @@ def plot_array_layout(
     grayed_out_elements=None,
     highlighted_elements=None,
     legend_location="best",
+    bounds_mode: str = "exact",
+    padding: float = 0.1,
+    x_lim: tuple[float, float] | None = None,
```

Comment:
Didn't you say you don't like type hinting?

Reply:
Fixed (this was the AI...)